spec/controllers/v2/editions_controller_spec.rb

```diff
@@ -124,6 +124,27 @@ def u(string)
       end
     end
 
+    context "filtered so that the result set is empty" do
+      it "returns an empty set" do
+        get :index, params: { locale: "nowhere" }
+        expect(parsed_response["results"].count).to eq(0)
+        expect(parsed_response["links"]).to eq([
+                                                 { "href" => "http://test.host/v2/editions?locale=nowhere", "rel" => "self" },
+                                               ])
+      end
+    end
+
+    context "filtered by document type" do
+      it "returns only the document type specified" do
+        create(:draft_edition, document_type: 'taxon', id: 10000)
```

Comment:
It might be good here to create something which isn't a taxon, just to make sure that the filtering is actually happening.

Reply:
There are 150 other documents which have already been created in the before(:context) block